launch_ros/launch_ros/actions/lifecycle_node.py

```diff
@@ -43,6 +39,7 @@ def __init__(
         *,
         name: SomeSubstitutionsType,
         namespace: SomeSubstitutionsType,
+        autostart: Optional[bool] = False,
```

Comment:
I think a docstring entry about this parameter here would be helpful. It's not clear to me why it's optional and what happens when it's `None`.

Reply:
Done for docstring, I remove None optionality